src/food.js

```diff
@@ -0,0 +1,37 @@
+export default function foodMenu() {
+
+    const products = ['HalwaPuri', 'Biryani', 'Chicken Karahi', 'Lassi', 'Drinks', 'Desserts']
+    const productsUrls = ['images/HalwaPuri.jpeg', 'images/Biryani.jpeg', 'images/Chicken.jpeg', 'images/Lassi.jpg', 'images/Drinks.jpg', 'images/dessert.jpeg']
```

Comment:
![Screenshot at 2020-07-05 14:04:17](https://user-images.githubusercontent.com/8848104/86533368-8f971000-bec8-11ea-9717-94cb5ee06e4d.png)
This error is due to the wrong name of images. Kindly fix it.